tooling/nargo_cli/tests/execution_success/hash_to_field/src/main.nr

```diff
@@ -1,5 +1,12 @@
 use dep::std;
 
 fn main(input : Field) -> pub Field {
-    std::hash::hash_to_field([input])
-}
+    let expected = std::hash::hash_to_field([input]);
+
+    let input_bytes = input.to_le_bytes(32);
+    let blake2s = std::hash::blake2s(input_bytes);
+    let got = dep::std::field::bytes32_to_field(blake2s);
```

Comment:
There are two further PRs to land;

- One to replace hash_to_field with this code (Should not be breaking as this test shows)
- Remove the hash_to_field blackbox function (Breaking change)

For the first point, we need to create an array that has size N * 32, since each field to hash_to_field needs to be converted into a 32 byte array

Reply:
First point is #3339